test/controllers/customers_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require "test_helper"
+
+class CustomersControllerTest < ActionDispatch::IntegrationTest
+  setup do
+    @customer = customers(:one)
```

Comment:
You can use a lets block here.
https://medium.com/@tomkadwill/all-about-rspec-let-a3b642e08d39